<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelNumeroDept">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Donnez un numéro de département :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumeroDept">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonObtenirNomDept">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Avoir le nom du département</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomDept">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du departement :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNomDept">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRegions">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDepartements">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxVilles">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
